src/Html/MutableAttributesTrait.php

```diff
@@ -75,4 +75,17 @@ public function createTag($tagName) {
     return $this->attributes->createTag($tagName);
   }
 
+  /**
+   * @param $key
+   * @param $value
+   *
+   * @return $this
+   *
+   * @see AttributesTrait::setAttribute()
+   */
+  public function setAttribute($key, $value) {
+    $this->attributes = $this->attributes->setAttribute($key, $value);
+    return $this;
+  }
+
```

Comment:
Seems ok to me.
The extra layer seems overly complex, but this is not your fault. And I have spent some brain juice designing this, so there may be good reasons.